### ISSUE TITLE
Unsynced Plugin Settings

### ISSUE THREAD

**Issue by erasedisknow** [runelite/runelite#11735]
So, I have two computers I play OSRS via RuneLite on, my main desktop PC running Windows 10 and my Laptop, which is a 2014 Macbook Pro I slapped Linux on. I keep my Runelite account logged in to both of them to keep the vast majority of my settings synced across both computers. However, there's a few plugins which currently do not seem to work right on Linux, namely combining GPU and Stretched Mode. I think a feature where we could set specific plugin settings to *NOT* sync with our accounts if we don't want them to would be great for people who have multiple computers, since currently, with syncing, if one computer doesn't support Plugin X (Almost always GPU) or if the displays are such that one computer warrants using stretched mode and the other doesn't, then the plugins Sync weirdly, like GPU getting turned off on the computer that supports it, even though the user may want it to always be on.

TL;DR: The ability to choose certain plugins to NOT sync the on/off setting of when switching computers would be a nice feature for people who have both a laptop and a desktop, or otherwise use multiple computers.

**Comment by Nightfirecat** [comment]
Duplicate of #8976
